docs/Integrating-with-Finch/Integration-Guides/Deel/Setup-Guide.md

```diff
@@ -0,0 +1,86 @@
+---
+stoplight-id: zz1jt69h6o6be
+---
+
+# Deel Onboarding Guide
+
+![deel-setup-diagram.png](<../../../../assets/images/deelSetupDiagram.png>)
+
+Integrating with Deel through Finch is a simple 3-step process! To get set up, you need to:
+
+1. Create a Deel account
+1. Create a Deel OAuth application
+1. Connect your Deel OAuth application to Finch
+
+This page explains each of these steps in detail. Happy integrating!
+
+> Integrating Deel via Finch makes your organization eligible for the Deel Partner Program. This includes go-to-market motions and revenue share opportunities with Deel. If you are interested or have already built an app, reach out here to learn more about the program.
+
+## 1. Create a Deel account
+
+If your company does not already have an account with Deel, you’ll need to create one. To create an account, follow the instructions below.
+
+1. Navigate to https://app.deel.com/signup in your browser
+1. Select the **I’m A Business** option and click **Next**
+
+    ![deel_signup_1.png](../../../../assets/images/deel_signup_1.png)
```

Comment:
you can use
```
<!--
focus: false
-->
```
if you want to get rid of the border/focus functionality on the images

Reply:
e.g.
```
<!--
focus: false
-->
![](../../assets/images/howFinchWorks4.png)
```